src/history/hs_conn.c

```diff
@@ -113,6 +113,9 @@ __wt_hs_config(WT_SESSION_IMPL *session, const char **cfg)
     btree->file_max = (uint64_t)cval.val;
     WT_STAT_CONN_SET(session, cache_hs_ondisk_max, btree->file_max);
 
+    /* To access the history store file later, set up the connection with the HS open flag. */
```

Comment:
Minor suggestion to the comment. I think it might be clearer to add or change to say that if we now have a handle to the btree we know the history store file is open and we can set the flag.

Reply:
Updated with a better comment to include the handle to the history store and file open.